packages/typegen/src/templates/lookup/defs.hbs

```diff
@@ -2,8 +2,6 @@
 
 /* eslint-disable sort-keys */
 
-import type { Definitions } from '../../types';
```

Comment:
I removed this import because the definitions are type-asserted on usage anyway: https://github.com/polkadot-js/api/blob/master/packages/typegen/src/fromDefs.ts#L29